src/doc/reference.md

```diff
@@ -1047,11 +1047,8 @@ This is a list of behavior not considered *unsafe* in Rust terms, but that may
 be undesired.
 
 * Deadlocks
-* Reading data from private fields (`std::repr`)
```

Comment:
How does this interact with what it says 18 lines previously?

>  [Behavior considered undefined] Invalid values in primitive types, even in private fields/locals

Is it now ok, since private fields are not permitted to be read?